projects/ion/src/lib/indicator/indicator.component.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { render, screen } from '@testing-library/angular';
+import { IonIconComponent } from './../icon/icon.component';
+
+import { IndicatorComponent, IndicatorProps } from './indicator.component';
+
+const defaultProps = {
+  label: 'Recuperável',
+  iconbody: 'technical',
+  type: 'info-solid',
+  value: '2.800',
+  color: '#6e7192',
+  percent: '30',
+};
+
+const sut = async (props: IndicatorProps = defaultProps) => {
```

Comment:
```suggestion
const sut = async (props: IndicatorProps = defaultProps): void => {
```
Missing return type on function